### DISCUSSION TITLE
I don't quit understand what index is in terns of blocks

### DISCUSSION THREAD

**Discussion by Tusenka** [prometheus/prometheus#15123] [Q&A]
Index Disk Format
When the index is written, an arbitrary number of padding bytes may be added between the lined out main sections above
https://github.com/prometheus/prometheus/blob/main/tsdb/docs/format/index.md

**Comment by bboreham** [comment]
Ganesh's blog is one of the best sources to learn about blocks and indexes.
https://ganeshvernekar.com/blog/prometheus-tsdb-persistent-block-and-its-index
